deploy/cdk/bin/base.ts

```diff
@@ -99,18 +100,33 @@ new imageProcessing.DeploymentPipelineStack(app, `${namespace}-image-processing-
   contact,
   namespace,
   ...imageProcessingProps,
-});
+})
+
+
 const elasticsearchContext = getContextByNamespace('elasticsearch')
 const elasticsearchProps = {
   namespace,
   foundationStack,
   ...elasticsearchContext,
 }
-new elasticsearch.ElasticStack(app, `${namespace}-elastic`, elasticsearchProps);
+new elasticsearch.ElasticStack(app, `${namespace}-elastic`, elasticsearchProps)
 new elasticsearch.DeploymentPipelineStack(app, `${namespace}-elastic-deployment`, {
   oauthTokenPath,
   owner,
   contact,
-  ...elasticsearchProps
+  ...elasticsearchProps,
+})
+
+const manifestPipelineContext = getContextByNamespace('manifestPipeline')
+new manifestPipeline.ManifestPipelineStack(app, `${namespace}-manifest-${envName}`, {
```

Comment:
This was how they were named before.

Also, once the test stack is deployed in testlib, if someone manually approves the deploy in testlib, the prod is also deployed in testlib.

Separate from that, when test is deployed in libnd, if someone approves that (either through slack or manually), the prod is then deployed in libnd.

Reply:
That's what I thought you were going for - I don't think this will quite replicate that. In this case, we are using `envName` as the AWS account environment, rather than a pipeline's stage. So your stacks for both stages of the pipeline would end up being the same: `marble-manifest-dev` in testlib, and `marble-manifest-prod`.

The other stacks pass the stage as part of the namespace, but that would produce `marble-test-manifest-dev` and `marble-test-manifest-prod`.

If you want the stage after the rest of the stack name, you'll need a different context variable. Then the pipeline can pass the appropriate stage name in, and you'll have to remember to provide both -c env and -c stage

Reply:
I made these recommended changes.  Thanks.